packages/@n8n/config/src/configs/insights.config.ts

```diff
@@ -0,0 +1,11 @@
+import { Config, Env } from '../decorators';
+
+@Config
+export class InsightsConfig {
```

Comment:
we don't need to keep this here, we can move this in the `insights` folder. that way everything specific to this module is isolated away in a single folder.

Reply:
But that would mean that @n8n/config would have  "n8n" (cli) as a dependency, which seems like something we do not want, no ? 
Or do you mean in an "insights" folder inside @n8n/config/src/configs ?